src/modules/85-cv/utils/CommonUtils.ts

```diff
@@ -156,6 +156,16 @@ export const getCVMonitoringServicesSearchParam = (props: GetCVMonitoringService
   )
 }
 
+export function getSearchString(params: { [key: string]: any }): string {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232283)
```suggestion
export function getSearchString(params: { [key: string]: unknown }): string {
```